### PR TITLE
Added "how to build web3" developer resources

### PR DESCRIPTION
**Pull request by 0xolishell**
I added the following popular "how to learn web3 dev" resources and courses (all free) - Alchemy University, Chainlink (specifically Patrick Collins' famous 32 hour comprehensive course / video), and Nader Dabit. In full transparency, I work for Alchemy and am submitting this list because these resources consistently receive the most positive feedback from both the crypto community and our developers. Love what you guys do at Ethereum, thank you!

### CODE REVIEW

File: src/content/community/support/index.md [hunk 1]
```diff
@@ -38,8 +38,11 @@ Looking for an Ethereum wallet? [Explore our full list of Ethereum wallets](/wal
 
 Building can be hard. Here are some development focused spaces with experienced Ethereum developers that are happy to help.
 
+- [Alchemy University](https://university.alchemy.com/#starter_code)
+- [Chainlink](https://blog.chain.link/learn-blockchain-full-stack-web3-javascript-smart-contract-development/)
 - [CryptoDevs discord](https://discord.gg/Z9TA39m8Yu)
 - [Ethereum StackExchange](https://ethereum.stackexchange.com/)
+- [Nader Dabit](https://naderdabit.notion.site/Nader-s-web3-Resources-for-Developers)
```
Comment (corwintines) [inline src/content/community/support/index.md:45]:
```suggestion
```
I cant access this, seems to be restricted

File: src/content/community/support/index.md [hunk 2]
```diff
@@ -38,8 +38,11 @@ Looking for an Ethereum wallet? [Explore our full list of Ethereum wallets](/wal
 
 Building can be hard. Here are some development focused spaces with experienced Ethereum developers that are happy to help.
 
+- [Alchemy University](https://university.alchemy.com/#starter_code)
+- [Chainlink](https://blog.chain.link/learn-blockchain-full-stack-web3-javascript-smart-contract-development/)
```
Comment (corwintines) [inline src/content/community/support/index.md:42]:
```suggestion
```

I'm not sure this belongs in this section of ethereum.org. We link out to this in the tutorials section, but my gut take is that a tutorial is not the same as an online community so this should be removed.

File: src/content/community/support/index.md [hunk 3]
```diff
@@ -38,8 +38,11 @@ Looking for an Ethereum wallet? [Explore our full list of Ethereum wallets](/wal
 
 Building can be hard. Here are some development focused spaces with experienced Ethereum developers that are happy to help.
 
+- [Alchemy University](https://university.alchemy.com/#starter_code)
```
Comment (corwintines) [inline src/content/community/support/index.md:41]: This looks fine as a link to me. I am just wondering out of curiosity, how is the Discords managed for Alchemy University? I see you need an NFT for access to the channel, is this free?

**Review by corwintines — COMMENTED** [review]
Just a couple questions about Alchemy University, and comments on the other links.

File: src/content/community/support/index.md (hunk 1 above)
Comment (corwintines) [reply src/content/community/support/index.md:45]: Would developer dao maybe be more appropriate? I know Nader is quite involved with that community

File: src/content/community/support/index.md (hunk 3 above)
Comment (shelleyolivia) [reply src/content/community/support/index.md:41]: Great question! Re: Discord - we have an education team (eng, product, devrel, etc) at Alchemy that created Alchemy University and helps man the Discord along with some of our community ambassadors. Re: free - Alchemy University like all our dev content is completely free - the NFT drops were to give people early access when the course launches in a few weeks. We have a waitlist of tens of thousands which has been so awesome to see. Lmk if you have any more questions! We are definitely doing a lot of Ethereum specific content so if there's any way we can be helpful to Ethereum too, seriously pls just lmk :)

File: src/content/community/support/index.md (hunk 2 above)
Comment (shelleyolivia) [reply src/content/community/support/index.md:42]: Hey Corwin - gotcha thanks, that makes sense, I'll defer to you! I didn't want to just link Alchemy docs but more so, resources that would benefits everyone. Just for context FWIW, we just see a ton of traffic and developers mentioning Patrick Collins specifically (Chainlink devrel) who is a force (in an awesome way) for the web3 dev space - he does a infamous 32 hour web3 dev Youtube video haha, that's the video on this Chainlink page.

File: src/content/community/support/index.md (hunk 1 above)
Comment (shelleyolivia) [reply src/content/community/support/index.md:44]: Thanks for flagging Corwin! Maybe with his recent move to Aave / Lens Protocol, some view settings changed? Does this work at all - https://naderdabit.notion.site/Nader-s-web3-Resources-for-Developers-a200ed2ef21c4d578dc158df2b882c63? I'm seeing it on my end in incognito too. 

If not, sure developer dao could be good. Maybe this one that looks more general? https://blog.developerdao.com/free-resources-to-learn-solidity-ethereum-blockchain-fullstack-dapps-and-smart-contract.

**Review by corwintines — APPROVED** [review]

**Pull request merged**